pkg/collector/jobevent_watcher.go

```diff
@@ -23,10 +23,21 @@ import (
 
 var watchStartTime = meta_v1.Now()
 
+var podFilter *regexp.Regexp
+
 type EventHandler struct {
 	collection *CronJobCollection
 }
 
+func createPodFilter() *regexp.Regexp {
```

Comment:
This is a bit confusing. It seems the only usage of this function is to assign to the global, in which case this should just return nothing. 

That said I much prefer it returning `*regexp.Regexp` and not having that `podFilter` global variable.